### DISCUSSION TITLE
reboot not working

### DISCUSSION THREAD

**Discussion by leeablett** [roleoroleo/yi-hack-MStar#398] [General]
Hi there,

For some reason, my reboot entry is not working and I can see it in the configuration.

/sbin # cat /var/spool/cron/crontabs/root
0 1 * * * /sbin/reboot
0 * * * * /home/yi-hack/script/clean_records.sh 50

Is my command ok?

0 1 * * * /sbin/reboot

**Comment by roleoroleo** [comment]
Use `reboot -f`
Something cancels the procedure during normal reboot.

**Reply by roleoroleo** [reply]
Maybe with some `sync` first.

**Reply by leeablett** [reply]
I will give that a go and I just want to say thank you for all your effort in making this project a success!

**Reply by leeablett** [reply]
That worked and thanks again!
